typings/index.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference path="globals/es6-shim/index.d.ts" />
```

Comment:
This is the "bundle file" that describes all the typings in this project (currently just this one). Used by IDEs to resolve "intellisense"... not compiled that's why it looks like a comment. See https://github.com/typings/typings/blob/master/docs/faq.md (the module resolutions section).

Also, if it helps this file is created by default if you do a clean "ionic start myApp --v2 --ts".